docs/how-to-use-relational-backend-storage.md

```diff
@@ -87,6 +91,81 @@ Finally, you can run the script in the distribution package directory to start t
 ./${GRAVITINO_HOME}/bin/gravitino.sh start
 ```
 
-## How to use H2
+## How to use PostgreSQL
+
+### Prerequisites
+
+- PostgreSQL 12 ~ 16. For other versions, the community has not tested fully and may not be compatible.
+- Gravitino distribution package with version `0.6.0` or above.
```

Comment:
Do we already support this feature in version 0.6.0?

Reply:
Sorry, it's a mistake and Gravitino only support it from 0.7.0.